.spi.yml

```diff
@@ -0,0 +1,22 @@
+version: 1
+external_links:
+  documentation: 'https://aws.amazon.com/sdk-for-swift'
```

Comment:
Given our current documentation is so out of date, should we remove this for now. We can add it back in after we update our docs.